package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Scaffolding for projects at Fullstack Academy",
   "main": "server/start.js",
   "scripts": {
-    "start": "node server/start.js",
+    "start": "nodemon server/start.js",
```

Comment:
i would keep it with node, and make a npm dev to use nodemon (it's nice to use node vs nodemon when you wnat to stay authorized)